### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Astouline**
Add name and pitch

### CODE REVIEW

**Review by benoit1521 — APPROVED** [review]

**Review by quentinsd — COMMENTED** [review]
ok

**Review by CorentinReims — APPROVED** [review]

**Review by quentinsd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# reims-0918-js--wall
\ No newline at end of file
+# reims-0918-js--wall
+
```
Comment (dadtmt) [inline README.md:2]: use md syntax to specify titles Team and Pitch (Pitch should come first)

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+Developers:
+
+Corentin de Soto Cobet
```
Comment (dadtmt) [inline README.md]: use * or - for list (check md syntax)

**Review by dadtmt — CHANGES_REQUESTED** [review]
If you want a nice Readme you should use Markdown Syntax

**Review by dadtmt — APPROVED** [review]

**Pull request merged**